### PR TITLE
Add all Roboto Flex axes except YTFI 

### PR DESCRIPTION
**Pull request by rsheeter**
Exploratory addition of axes for Roboto Flex. Don't stress the descriptions too much as these are just to experiment with for now.

YTFI is not defined in https://variationsguide.typenetwork.com/

### CODE REVIEW

File: axisregistry/x_opaque.textproto [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# XOPQ based on https://variationsguide.typenetwork.com/#xopq
+tag: "XOPQ"
+display_name: "Y transparent"
+min_value: -1000
+max_value: 2000
+default_value: 88
+precision: 0
+fallback {
```
Comment (davelab6) [inline axisregistry/x_opaque.textproto:8]: Is fallback required? If so, using the default values seems reasonable. 

Like opsz, parametric axes defaults don't make sense as library-wide/global values, they are all family specific. 

Like Slant, parametric axes have no meaningful global fallbacks.

File: axisregistry/y_opaque.textproto [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# YOPQ based on https://variationsguide.typenetwork.com/#yopq
+tag: "YOPQ"
+display_name: "Y transparent"
```
Comment (davelab6) [inline axisregistry/y_opaque.textproto:3]: "Y opaque"

File: axisregistry/y_transparent_lowercase.textproto [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# YTLC based on https://variationsguide.typenetwork.com/#ytlc
+tag: "YTLC"
+display_name: "Y transparent"
```
Comment (davelab6) [inline axisregistry/y_transparent_lowercase.textproto:3]: "Y transparent lowercase"

File: axisregistry/x_opaque.textproto (hunk 1 above)
Comment (rsheeter) [reply axisregistry/x_opaque.textproto:8]: IIRC you have to have at least one fallback so we know what you get on a variable incapable UA

File: axisregistry/x_opaque.textproto [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# XOPQ based on https://variationsguide.typenetwork.com/#xopq
+tag: "XOPQ"
+display_name: "Y transparent"
```
Comment (davelab6) [inline axisregistry/x_opaque.textproto:3]: "X opaque"

**Comment by davelab6** [comment]
> YTFI is not defined in variationsguide.typenetwork.com

I've asked @dberlow to provide this, and update that site with it, but here's an initial draft inline with what you have:

```
# YTFI based on https://variationsguide.typenetwork.com/#xtfi
tag: "XTFI"
display_name: "X transparent figures"
min_value: -1000
max_value: 2000
default_value: 400
precision: 0
fallback {
  name: "Normal"
  value: 400
}
description:
  "Assigns a 'white' per mille value to each instance of the design space."
```

**Review by davelab6 — COMMENTED** [review]
Please update the display names and add the XTFI axis

File: axisregistry/y_opaque.textproto (hunk 2 above)
Comment (rsheeter) [reply axisregistry/y_opaque.textproto:3]: Fixing on main since this is merged

File: axisregistry/y_transparent_lowercase.textproto (hunk 3 above)
Comment (rsheeter) [reply axisregistry/y_transparent_lowercase.textproto:3]: Fixing on main since this is merged

**Comment by rsheeter** [comment]
> Please update the display names and add the XTFI axis

Done

File: axisregistry/x_opaque.textproto (hunk 1 above)
Comment (davelab6) [reply axisregistry/x_opaque.textproto:8]: Can those be family specific? I also forget...

**Pull request merged**